src/linkAttributes/livePreview.ts

```diff
@@ -130,26 +129,7 @@ export function buildCMViewPlugin(app: App, _settings: SuperchargedLinksSettings
                                             widget: new HeaderWidget(attributes, true),
                                         });
 
-                                        if (isMDUrl) {
```

Comment:
A lot of code is removed here. Can you describe what functionality it is that was removed here and if this may affect existing users somehow?

Reply:
This fixes #185. "isMDUrl" checks for `[links](like this)` (not `[[like this]]`). Normally you'd use the first format for external links (`[Example](https://example.com)`), but to be fair, you can also link internally like that (`[Example](Note Title)`).
Some preliminary testing with using this format for internal links does reveal weird behaviour of the plugin though (e.g. styles on other links break).
I just took out all code that deals with these kinds of links. Now they are completely ignored.